howto/faq/README.md

```diff
@@ -45,7 +45,12 @@ We learn about upcoming API breakages in UIKit as early as everyone else through
 
 ### How large is the upload size of the library?
 
-If you import all the components, the size of your app will increase by about 5.3MB. However, you don't have to add the entire library! You can add as many or as few of the 25+ individual components as you like. And that averages to less than .2MB per component.
+The size of different components vary from a just a few [kibibytes (1 KiB = 1024
```

Comment:
Love it! But you don't need to include the definitions of KB and MB and links. We can expect our audience is technically proficient.

Reply:
Done